examples/jsm/renderers/webgpu/WebGPUUniformBuffer.js

```diff
@@ -38,6 +36,12 @@ class WebGPUUniformBuffer extends WebGPUBinding {
 
 	}
 
+	static getByteLength( byteLength ) {
```

Comment:
I would avoid putting static methods into such classes. Maybe introduce a utility class with a pure static interface instead? Something like `WebGPUBufferUtils.getByteLength();`?

@mrdoob What is your preference?

Reply:
Sounds good to me 👍